docs/examples/conda/README.ipynb

```diff
@@ -29,9 +29,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
:question: Are the examples supposed to show output?  Asking as it seems they were left blank previously